pkg/cmd/pipeline/describe_test.go

```diff
@@ -117,6 +117,47 @@ func TestPipelineDescribe_empty(t *testing.T) {
 	golden.Assert(t, got, fmt.Sprintf("%s.golden", t.Name()))
 }
 
+func TestPipelineDescribe_WithoutNameOfOnlyOnePipelinePresent(t *testing.T) {
+	clock := clockwork.NewFakeClock()
+
+	pipelines := []*v1alpha1.Pipeline{
+		tb.Pipeline("pipeline",
+			tb.PipelineNamespace("ns"),
+			// created  5 minutes back
```

Comment:
Please try to avoid the use of `tb` or test builders when writing unit tests as they will be deprecated in the future. You can use structs instead like you did in the `v1beta1` test.